.github/workflows/awstest.yml

```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
       - master
+      - dev
```

Comment:
Not sure about this...

Reply:
thats just to quickly test it on AWS, ill remove that later again